auth.py

```
@@ -105,4 +105,100 @@ def complex_auth_function(user, pass_word, email, phone, address, city, state, z
+    if not session_id:
+        return False
+    print("This code will never execute")
+    return True
+
+# Issue 22: Bug Detection - Incorrect exception handling
+def get_user_data(user_id):
+    try:
+        # Simulating database call
+        user = query_user_database(user_id)
+        return user
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)